### PR TITLE
Refactoring AT_01.05.004 and AT_01.05.005 implementation

### PR DESCRIPTION
**Pull request by Churs82**
I have merged  tests for one user case in one file with shared constants and functions

### CODE REVIEW

**Comment by Rom-T** [comment]
If you want to refactor something, please create a card in Trello for refactoring.

File: cypress/e2e/headerUserConfigure.cy.js [hunk 1]
```diff
@@ -1,11 +1,62 @@
 /// <reference types="cypress"/>
 
-describe('Header User Configure', () => {
-    
-    it('AT_1.05_001| <Header>User configure', function () {
+describe('US_01.05 | <Header>User configure', () => {
```
Comment (Rom-T) [inline cypress/e2e/headerUserConfigure.cy.js:3]: Please remove number and <> from the describe. 
'Header | User configure'

File: cypress/e2e/headerUserConfigure.cy.js [hunk 2]
```diff
@@ -1,11 +1,62 @@
 /// <reference types="cypress"/>
 
-describe('Header User Configure', () => {
-    
-    it('AT_1.05_001| <Header>User configure', function () {
+describe('US_01.05 | <Header>User configure', () => {
+    Cypress.Commands.add('navigateUserConfigurationPage', () => {
```
Comment (Rom-T) [inline cypress/e2e/headerUserConfigure.cy.js:4]: Usually such commands are created for global use and for wrapping them in sai, but here it is better to use a function. But this is not critical yet. We will refactor this later.

Comment (Churs82) [reply cypress/e2e/headerUserConfigure.cy.js:4]: Ok. I'll make local functions on native js in future scripts

**Review by ArtyomDulya — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Pull request merged**